qutip/qip/circuit.py

```diff
@@ -220,6 +220,8 @@ def add_gate(self, gate, targets=None, controls=None, arg_value=None,
             Argument value(phi).
         arg_label: String
             Label for gate representation.
+        index : List
```

Comment:
`list` not List. It is more about specifying the datatype than describing it.

Reply:
noticed this now. @abhisekupadhyaya if you can address it in a new PR, it is a quick fix.

Reply:
Sure. I will get it done soon.